postprocessing/M4aNoDash.java

```diff
@@ -0,0 +1,41 @@
+package us.shandian.giga.postprocessing;
+
+import org.schabi.newpipe.streams.Mp4DashReader;
+import org.schabi.newpipe.streams.Mp4FromDashWriter;
+import org.schabi.newpipe.streams.io.SharpStream;
+
+import java.io.IOException;
+
+class M4aNoDash extends Postprocessing {
+
+    M4aNoDash() {
+        super(false, true, ALGORITHM_M4A_NO_DASH);
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider making constructor public for access</b></div>
<div id="fix">

Consider making the constructor `public` since it needs to be accessible from other packages to be useful as a postprocessing implementation.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -11,2 +11,2 @@
 -    M4aNoDash() {
 +    public M4aNoDash() {
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/OBrutus/youtube-downloader/pull/9#issuecomment-2464377435>#3b9ab6</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged